internal/server/statvar_group.go

```diff
@@ -295,6 +296,9 @@ func (s *Server) GetStatVarPath(
 	for {
 		if parents, ok := s.cache.ParentSvg[curr]; ok {
 			curr = parents[0]
+			if curr == svgRoot {
```

Comment:
There will be a different PR for the current flow? (i.e., with full hierarchy showing up in the client)

Reply:
I am thinking to do that in mixer. So instead of relying on triples cache, mixer will read from the full hierarchy, and return the queried node.